src/commands/lighthouse/show.ts

```diff
@@ -0,0 +1,33 @@
+import { flags as oclifFlags } from '@oclif/command'
+
+import { CustomCommand } from '../../oclif/CustomCommand'
+import { showReports } from '../../modules/lighthouse/showReports'
+
+export default class ShowReports extends CustomCommand {
+  static description = 'Show previous saved audit reports, filtering by app and/or url'
+
+  static examples = [
+    'vtex lighthouse show -app=vtex.awesome-app',
+    'vtex lighthouse show -u https://awesome.store.com',
+    'vtex lighthouse show -a=vtex.awesome-app --url=https://awesome.store.com',
```

Comment:
Maybe when using the short versions we should prefer not to use `=`
```suggestion
    'vtex lighthouse show -a vtex.awesome-app --url=https://awesome.store.com',
```

Reply:
Totally agreed! It was not supposed to have a `=` there